common/components/SendButtonFactory/SendButtonFactory.tsx

```diff
@@ -19,7 +19,7 @@ interface StateProps {
 }
 interface OwnProps {
   onlyTransactionParameters?: boolean;
-  Modal: typeof ConfirmationModal;
+  Modal: typeof ConfirmationModalTemplate;
```

Comment:
This should be typeof ConfirmationModal where ConfirmationModal implements ConfirmationModalTemplate

Reply:
And also is not used anymore, breaking #940